feature-libs/product-configurator/rulebased/components/overview-menu/configurator-overview-menu.component.html

```diff
@@ -0,0 +1,111 @@
+<ng-container *ngIf="ovGroups$ | async as ovGroups; else ghostMenu">
+  <div class="cx-sticky-menu">
+    <div class="cx-menu-bar">
+      <button role="tab" class="active">
```

Comment:
Is there a reason for making the heading of the menu a button? There is no action that could be triggered.

Reply:
The reason is that soon there will be an action (switching between menu and filter component). This navigation will be moved to yet another component then, still I think we can leave this as it is (filter will be merged in 1 or 2 weeks)